x/community/keeper/keeper.go

```diff
@@ -31,12 +37,17 @@ func NewKeeper(ak types.AccountKeeper, bk types.BankKeeper, dk types.Distributio
 		panic("legacy community pool address not found")
 	}
 
-	return Keeper{
-		bankKeeper:    bk,
-		distrKeeper:   dk,
-		hardKeeper:    hk,
-		moduleAddress: addr,
+	if !params.HasKeyTable() {
```

Comment:
what scenarios would trigger this if block to happen / is that worth documenting for future developers with a comment?